go/runtime/bundle/manifest.go

```diff
@@ -226,6 +227,24 @@ type Identity struct {
 	Enclave sgx.EnclaveIdentity `json:"enclave"`
 }
 
+// ExplodedComponent is an exploded runtime component ready for execution.
```

Comment:
Should this be in `bundle.go` instead of `manifest.go`? To me it seems like that would make more sense.

Reply:
Agree. But maybe it is even clearer to move it to a component file. So I did that.